src/libexpr/flake/flake.hh

```diff
@@ -17,20 +17,41 @@ struct FlakeInput;
 
 typedef std::map<FlakeId, FlakeInput> FlakeInputs;
 
+// FlakeInput is the flake-level parsed form of the "input" entries in
```

Comment:
Could you use `/* ... */` style comments? We mostly use `//` for single-line comments.

Reply:
Sure.  I made this in its own commit with no other wording changes (although some `fmt` wrapping updates occurred); all wording changes are in different commits to make it easier to see those changes.